### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by daangroot**
## Description

This PR removes alot of dead/redundant code, mainly backend python code. It's no longer necessary to run with the `only-server` flag. @cecpk @sn0opy @na-ji maybe it's a good idea to create a backup in a seperate branch of the current dev, just in case...

## Motivation and Context

Since this fork of RocketMap is used as a front end for MAD, alot of backend code has become redundant.

## How Has This Been Tested?

Local instance.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by na-ji** [comment]
Wow, awesome work. :clap: I will test it tomorrow
I think we will merge (soon) develop in master and start a new develop with your PR

**Comment by daangroot** [comment]
@sn0opy The purpose of this PR is to clean up the code. All references to the MrMime module have already been removed. `account.py` and `captcha.py` have been deleted completely.

**Comment by sn0opy** [comment]
@daangroot totally my bad. I totally missed the part, that this was a commit to a seperate branch PR. Looked like a commit to the develop branch. Sorry.

File: config/config.ini.example [hunk 1]
```diff
@@ -60,86 +41,44 @@
...
-#hlvl-kph:                      # Set a maximum speed in km/hour for high level account scanning. 0 to disable. (default=25)
-#medalpokemon                   # Show notify for tiny rattata and big magikarp on map
+#medalpokemon                   # Notify of tiny rattata and big magikarp on map. (default=False)
 
 # Webserver settings
 ####################
 
-#only-server                    # Server-Only Mode
 #locale:                        # Pokemon translation
-#no-fixed-location              # Disables the fixed map location and shows the search bar for use in shared maps.
```
Comment (na-ji) [inline config/config.ini.example]: So now we know what it's used for, it could be added back? :upside_down_face:

Comment (cecpk) [reply config/config.ini.example]: 😀

Comment (daangroot) [reply config/config.ini.example]: It's back.

**Pull request merged**